bitops.go

```diff
@@ -0,0 +1,339 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"fmt"
+	"strings"
+	"bufio"
+	"log"
+)
+
+var lineBreak string = "#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#~#"
+var loopBreak string = "-------------------------------------"
+var confirmResponse string = "Is this correct?"
+var helpResponse string = `
+BitOps:
+  init 		- Initializes a new Operations Repo
+  add-new	- Add a new environment or tool to an existing Operations Repo
+`
+var supportPlugins []string = []string{"aws", "cloudformation", "terraform", "ansible", "helm"}
```

Comment:
These vars should be constants https://go.dev/tour/basics/15

Reply:
```suggestion
const SupportedPlugins []string = []string{"aws", "cloudformation", "terraform", "ansible", "helm"}
```
If using constants, they're CamelCase starting with the uppercase.